tests/basic/customer_issues_test.go

```diff
@@ -994,3 +998,200 @@ var _ = Describe("{ContainerCreateDeviceRemoval}", func() {
+	command := fmt.Sprintf("iptables -A INPUT -p tcp -s %v --dport 3260 -j DROP ", n.MgmtIp)
+	if !block {
+		command = fmt.Sprintf("iptables -D INPUT -p tcp -s %v --dport 3260 -j DROP", n.MgmtIp)
+	}
+	log.InfoD("Triggering command [%s] from Node [%v]", command, n.Name)
+	err := runCommand(command, *n)
+	if err != nil {
+		return err
+	}
+
+	// Block all Outgoing requests from port 3260 from specific Node
+	command = fmt.Sprintf("iptables -A OUTPUT -p tcp -s %v --dport 3260 -j DROP ", n.MgmtIp)
+	if !block {
+		command = fmt.Sprintf("iptables -D OUTPUT -p tcp -s %v --dport 3260 -j DROP", n.MgmtIp)
+	}
+	log.InfoD("Triggering command [%s] from Node [%v]", command, n.Name)
+	err = runCommand(command, *n)
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+// getPVCAccessMode returns list of Accessmodes for the PVCs
+func getPVCAccessMode(pvcName string, pvcNameSpace string) ([]corev1.PersistentVolumeAccessMode, error) {
+	accessModes := []corev1.PersistentVolumeAccessMode{}
+	pvc, err := k8sCore.GetPersistentVolumeClaim(pvcName, pvcNameSpace)
+	if err != nil {
+		return nil, err
+	}
+
+	for _, mode := range pvc.Spec.AccessModes {
+		log.Infof("Access Mode [%v]", mode)
+		accessModes = append(accessModes, mode)
+	}
+
+	return accessModes, nil
+}
+
+// Flush all IPTable Rules from all the Nodes Created
+func flushAllIPtableRulesOnAllNodes() {
+	for _, eachNode := range node.GetNodes() {
+		command := "iptables -F"
+		if !node.IsMasterNode(eachNode) {
+			err := runCommand(command, eachNode)
+			if err != nil {
+				log.FailOnError(err, "Failed to flush iptable rules")
+			}
+		}
+	}
+}
+
+var _ = Describe("{FADAPodRecoveryAfterBounce}", func() {
+
+	/*
+				PTX : https://purestorage.atlassian.net/browse/PWX-31647
+			Test to check if the Pod bounces and Places in the new node when iscsi port fails to Send / Receive Traffic
+			Specific to FADA Volumes with RWO type
+
+		Px Implementation :
+			A background task will periodically look for any readonly FADA volumes and those pods will be bounced.
+			The default period is 15 seconds and this period is configurable through a cluster option called ro-vol-pod-bounce-interval
+			If a volume was expected to be read-only during creation (e.g ReadOnlyMany PVCs), it’ll be excluded.
+
+	*/
+	JustBeforeEach(func() {
+		StartTorpedoTest("FADAPodRecoveryAfterBounce",
+			"Verify Pod Recovers from RO mode after Bounce",
+			nil, 0)
+	})
+	var contexts []*scheduler.Context
+	stepLog := "App Stuck in ContainerCreation State with Device Exists in the backend"
+	It(stepLog, func() {
+
+		contexts = make([]*scheduler.Context, 0)
+		for i := 0; i < Inst().GlobalScaleFactor; i++ {
+			contexts = append(contexts, ScheduleApplications(fmt.Sprintf("fapodrecovery-%d", i))...)
+		}
+		ValidateApplications(contexts)
+		defer DestroyApps(contexts, nil)
+		defer flushAllIPtableRulesOnAllNodes()
+
+		// Wait for 2 min for pods to get settle down before proceeding further
+		time.Sleep(2 * time.Minute)
```

Comment:
app validation will make sure all apps are running. I think no need of 2 min wait here